src/utils/__tests__/satisfyVersion.spec.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+import { satisfyVersion } from '../satisfyVersion.ts'
+
+describe('satisfyVersion', () => {
+	const testCases = [
+		// Older
```

Comment:
```suggestion
		// Newer
```

Reply:
for me `older` is what was published before (like old). maybe, pick something more definitive?